examples/04_glm_first_level/plot_write_events_file.py

```diff
@@ -24,41 +25,193 @@
 # example.
 import numpy as np
 
-onsets = np.array([
-    0., 2.4, 8.7, 11.4, 15., 18., 20.7, 23.7, 26.7, 29.7, 33., 35.4, 39.,
-    41.7, 44.7, 48., 56.4, 59.7, 62.4, 69., 71.4, 75., 83.4, 87., 89.7,
-    96., 108., 116.7, 119.4, 122.7, 125.4, 131.4, 135., 137.7, 140.4,
-    143.4, 146.7, 149.4, 153., 156., 159., 162., 164.4, 167.7, 170.4,
-    173.7, 176.7, 188.4, 191.7, 195., 198., 201., 203.7, 207., 210.,
-    212.7, 215.7, 218.7, 221.4, 224.7, 227.7, 230.7, 234., 236.7, 246.,
-    248.4, 251.7, 254.7, 257.4, 260.4, 264., 266.7, 269.7, 275.4, 278.4,
-    284.4, 288., 291., 293.4, 296.7,
-])
+onsets = np.array(
```

Comment:
Maybe we can refactor this and the next array or leave as is (before the reformatting)?
Not sure why they have to be hardcoded. We can instead generate them with a numpy function and print them to showcase what they look like

Reply:
I haven't looked too much into it, but the pattern isn't that obvious to me... 🤔 
A possible alternative, could be to declare it as a string which would be easier to format in a pretty way:

```python
onset_data = "[0.0, 2.4, 8.7, 11.4, 15.0, 18.0, 20.7, 23.7, 26.7]"
onsets = np.array(json.loads(txt))
```

Reply:
went with your option

just afraid it may confuse readers

another option make the number of events shorter: say 2 per condition

Reply:
I know we want to avoid this as much possible but this could be a good case for using `# fmt: off` no? I think when it comes to user facing examples we need to worry more about readability and being understandable/simple than when dealing with the codebase.

Reply:
yeah I was thinking about this as well

I think for the examples it does make sense to show pythonic code but make sure it is readable too.

let me test it

Reply:
Great let's go with then when needed for examples